scss/_utilities_layout.scss

```diff
@@ -4,4 +4,14 @@
   .u-fixed-width {
     @extend %fixed-width-container;
   }
+
+  @if ($table-layout != 'fixed') {
```

Comment:
This conditional is very difficult to document and can lead to unexpected regressions. If someone isn't aware of all the uses of these utilities and changes the variable in settings. This utility will disappear and things will regress.

Reply:
My idea here was to keep the generated css as minimal as possible. What this rule does is, if the base rule is set to 
```table-layout: auto;```

then you don't need a utility that overrides this with the same value. You might however need a utility that overrides it with 'fixed', and viceversa.

No if you change $table-layout from `auto` to `fixed`, any `.u-table-layout--fixed` utilities in the markup are no longer necessary because this is the default. So we don't generate rules for that class. Any references to this utility in the markup would simply be ignored.

So it won't lead to a regression, just cleaner css and less cascade overrides.